content/en/agent/kubernetes/apm.md

```diff
@@ -17,25 +17,32 @@ To enable trace collection with your Agent, follow the instructions below:
 
 1. **Configure the Datadog Agent to accept traces**:
 
-    {{< tabs >}}
-{{% tab "Daemonset" %}}
+{{< tabs >}}
+{{% tab "Helm" %}}
 
-To enable APM trace collection in Kubernetes:
+**Note**: If you want to deploy the Datadog Agent as a deployment instead of a DaemonSet, configuration of APM via Helm is not supported.
```

Comment:
Only the Cluster Agent would be deployed as a Deployment. For the Datadog Agent, this use case is very rare. Should we remove it then?

Reply:
@l0k0ms ?

Reply:
Gus is on vacation this week, so I'm covering for him. Let's remove for now. I'll add it as a suggestion.